Dockerfiles/mariadb/mariadb.conf

```diff
@@ -0,0 +1,17 @@
+[mariadb]
+# File Key Management
+plugin_load_add = file_key_management
+file_key_management_filename = /etc/mysql/encrypt/keys.enc
+file_key_management_filekey = FILE:/etc/mysql/encrypt/pass.key
+file_key_management_encryption_algorithm = AES_CBC
```

Comment:
The recommended one is stated as AES_CTR. Is there a reason that prevents us from using it?

Reply:
Nothing apart from my reading comprehension.